cli/package.json

```diff
@@ -31,7 +31,7 @@
     "dev": "tsup --watch",
     "clean": "rm -rf dist .turbo node_modules",
     "start": "node dist/index.js",
-    "lint": "eslint src --cache --cache-strategy content",
```

Comment:
this cache doesn't do anything in CI since the file's not persisted. We're better off with turbo's remote cache